src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -31,7 +31,7 @@ public class RazorViewEngineOptions
         /// locate Razor files.
         /// </summary>
         /// <remarks>
-        /// At startup, this is initialized to include an instance of <see cref="PhysicalFileProvider"/> that is
+        /// At startup, this is initialized to include an instance of PhysicalFileProvider that is
```

Comment:
Might be ok - the most common issues I've seen are about users adding `EmbeddedFileProviders` not Physical. That said would be good to get feedback from @rynowak \ @Eilon who originally asked for this property.